assets/app/view/game/round/auction.rb

```diff
@@ -168,6 +171,58 @@ def render_input(company)
           h(:div, { style: { textAlign: 'center', margin: '1rem' } }, company_actions)
         end
 
+        def render_turn_bid
+          return if !@current_actions.include?('bid') || @step.auctioning != :turn
+
+          input = h(:input, style: { margin: '1rem 0px', marginRight: '1rem' }, props: {
+            value: @step.min_player_bid,
+            step: @step.min_increment,
+            min: @step.min_player_bid,
+            max: @step.max_player_bid(@current_entity),
+            type: 'number',
+            size: @current_entity.cash.to_s.size,
+          })
+
+          create_bid = lambda do
+            hide!
+            price = input.JS['elm'].JS['value'].to_i
+            process_action(Engine::Action::Bid.new(
+              @current_entity,
+              price: price,
+            ))
+            store(:selected_company, nil, skip: true)
```

Comment:
do you actually need this?

Reply:
I think so - to control which company/corporation gets to respond to a click.

Reply:
but when you're auctioning do you ever choose tho company / corp to auction? i thought you're bidding for rights

Reply:
I fixed my reply - it's a mistake.